### ISSUE TITLE
[tempo-distributed] Cannot create ingester without enabling zone-awareness

### ISSUE THREAD

**Issue by MicroOps-cn** [grafana/helm-charts#3202]
@KyriosGN0 In a submission last week, region awareness was added. If region awareness is not enabled, the container for the singer will not be created.
The problem is here, It seems that the `defaultZone` variable is not being used: 
https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/templates/ingester/_helpers-ingester.tpl#L5-L25

**Comment by KyriosGN0** [comment]
yes, i have been made aware of this issue, and i opened a PR to fix this 
https://github.com/grafana/helm-charts/pull/3189

**Comment by KyriosGN0** [comment]
@MicroOps-cn fix was merged, tested and it works
